gradle/libs.versions.toml

```diff
@@ -17,50 +17,58 @@
 # under the License.
 #
 [versions]
-pulsar = "3.1.0"
-junit-jupiter = "5.8.2"
-log4j = "2.18.0"
```

Comment:
At first glance I was concerned w/ the major and minor version updates, however, they are all in testing libs (except for log4j which is a minor update from `2.18.0` -> `2.22.0`).